.changeset/nervous-suns-love.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
I am not sure how this works / where it is consumed so I can't really speak to it

Reply:
Probably not a blocker though.

Reply:
This is consumed by our GitHub Actions workflow when we release a new version of the runtime